android/app/src/main/java/com/contacttexter/CommunicationsModule.java

```diff
@@ -36,9 +36,8 @@ public void createPhoneCall(String number) {
 
     @ReactMethod
     public void createSMSMessage(String number, String message) {
-        Intent smsIntent = new Intent(Intent.ACTION_VIEW);
-        smsIntent.setType("vnd.android-dir/mms-sms");
-        smsIntent.putExtra("address", number);
+        Uri uri = Uri.parse("smsto:" + number);
```

Comment:
Using the intent ACTION_SENDTO did the trick on my device running 6.0.1.  The documentation for this intent says it was introduced in API level 1, so it should be safe to use across other Android versions:

https://developer.android.com/reference/android/content/Intent.html#ACTION_SENDTO